### ISSUE TITLE
Reviewing and Updating the Requirements

### ISSUE THREAD

**Issue by yusuferdemnacar** [bounswe/bounswe2022group6#40]
The [requirements page](https://github.com/bounswe/bounswe2022group6/wiki/Requirements) needs to be reviewed and updated according to customer meeting 2 and meeting  6.

The following should be done:
* Contradicting items should be removed
* Ambiguous topics should be explained further
* A glossary needs to be added
* A table of contents should be added
* Dropdown menus should be removed

**Deadline:** 30.03.2022

**Comment by yusuferdemnacar** [comment]
I removed the reviewing task from the issue for the following reasons:
* The task of reviewing the requirements was already done in the meetings
* The issue should be about one task

**Comment by yusuferdemnacar** [comment]
I updated the [requirements](https://github.com/bounswe/bounswe2022group6/wiki/Requirements) accordingly, however, it is not done yet, I am sharing this so that reviews can be done in parallel.
For now, the following topics are not yet completed:
* System requirements for messaging between users
* System requirements for Banning/Unbanning system

**Comment by yusuferdemnacar** [comment]
I added the missing parts mentioned in [comment #2](https://github.com/bounswe/bounswe2022group6/issues/40#issuecomment-1082539076). Updating is done. Please comment on any changes to the [requirements](https://github.com/bounswe/bounswe2022group6/wiki/Requirements) under the relevant issues.
